roles/app/templates/application.yml.j2

```diff
@@ -18,11 +18,18 @@ CHECKOUT_ZONE: {{ checkout_zone }}
 
 GOOGLE_MAPS_API_KEY: {{ google_maps_api_key }}
 
-{% if s3_backups_bucket is defined %}
-S3_BUCKET: {{ s3_backups_bucket }}
+{% if s3_access_key is defined %}
 S3_ACCESS_KEY: {{ s3_access_key }}
+{% endif %}
+{% if s3_access_key is defined %}
```

Comment:
I guess you meant `s3_secret` here. Otherwise we can merge the block with previous conditional.